build.sbt

```diff
@@ -45,9 +45,10 @@ lazy val interopScalaz7x = crossProject(JSPlatform, JVMPlatform)
   .settings(buildInfoSettings)
   .settings(
     libraryDependencies ++= Seq(
-      "dev.zio"    %%% "zio"                       % "1.0.0-RC11-1",
+      "dev.zio"    %%% "zio"                       % "1.0.0-RC12-1",
```

Comment:
Wanna upgrade to RC13 while you're here? The title of the PR sounds like you wanted to 😄

Reply:
I was too quick. I'm not going to use zio-test, and yes, RC13